WordPress/Classes/Models/KeyringConnection.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+/**
+ KeyringConnection represents a keyring connected to a particular external service.
+ We only rarely need keyring data and we don't really need to persist it. For these
+ reasons KeyringConnection is treated like a model, even though it is not an NSManagedObject,
+ but also treated like it is a Remote Object.
+*/
+@objc public class KeyringConnection : NSObject
+{
+    public var additionalExternalUsers = NSArray()
+    public var dateIssued = NSDate()
+    public var dateExpires:NSDate? = nil
```

Comment:
I wish the syntax didn't look so different between... an assignment, and a type declaration. 
Maybe Swift 3 will bring... `= NSDate?`